tests/guid.spec.ts

```diff
@@ -1,35 +1,54 @@
-import { assert, expect } from "chai";
+import { expect } from "chai";
 import "mocha";
 import { Guid } from "../lib/guid";
 
-describe("Guid", () => {    
-    const exampleGuid: string = "0315642c-a069-9f3e-1852-9adf2d075b93";
+describe("Guid", () => {
+    const example: string = "0315642c-a069-9f3e-1852-9adf2d075b93";
+    const example_no_hyphen: string = "0315642ca0699f3e18529adf2d075b93";
 
-    it("should create & validate a random guid", () => {
+    it("should create & validate a random GUID", () => {
         const wrong: string = "wrongguid";
         expect(Guid.isValid(wrong)).equal(false);
 
-        const right: Guid = Guid.create();
-        expect(Guid.isValid(right)).equal(true);
+        const statically: Guid = Guid.create();
```

Comment:
try to use more understandable variable names (e.g -> static_guid or guid_static)

```suggestion
        const static_guid: Guid = Guid.create();
```